db/migrate/20170109161435_move_lock_version_data.rb

```diff
@@ -0,0 +1,30 @@
+class MoveLockVersionData < ActiveRecord::Migration[5.0]
+  def up
+    execute "UPDATE link_sets
+             SET lock_version = t.number
+             FROM (SELECT target_id, number FROM lock_versions) t
+             WHERE link_sets.id = t.target_id"
+
+    execute "UPDATE documents
```

Comment:
Do you think if we try run a find_each and loop through each record this might be less of a strain? so tons of short queries rather than one slow one?